src/main/java/com/shippo/Shippo.java

```diff
@@ -9,6 +9,9 @@
     public static int RATES_REQ_TIMEOUT = 25000; //  milliseconds
     public static int TRANSACTION_REQ_TIMEOUT = 25000; // milliseconds
 
+    public static int httpConnectTimeout = 30 * 1000; // milliseconds
+    public static int httpReadTimeout = 80 * 1000; // milliseconds
+
     public static volatile String apiKey;
     public static volatile String apiVersion;
```

Comment:
noticing these names don't follow convention 😩  not something you need to address in this PR but it would be nice to carve out some time to clean up this SDK in the future (maybe a tech Wednesday?)

Reply:
Sounds good to me